### ISSUE TITLE
Subtract values swapped

### ISSUE THREAD

**Issue by Chorr0** [ChronoDK/GodotBigNumberClass#32]
When trying to make an upgrade system, I noticed that when I tried to buy an upgrade, if the cost was 10, and I had 100 money, eventually I could buy into the negative numbers. At first, I thought it was a problem with the isGreaterThanOrEqualTo function, as another problem suggested.

But, using a lot of prints to see what was going on, I noticed that when Subtracting, depending on the numbers of exponents of the values (generally if the money was 10 times or more than the cost), the value of the money's exponent would not go down, even if the mantissa was lower than 1

Long story short, I dug a lot and tried messing around with the isGreaterThanOrEqualTo, add and even the normalize functions. Finally, I tried swapping the "return add(x, negated_y)" in the subtract function with "return add(negated_y, x)" instead AND IT WORKS

please fix this when you are able to

**Comment by ondrejhonus** [comment]
I can't explain how much i love you bro

**Comment by Atovange** [comment]
Good catch, should definitely be merged @ChronoDK
